css-colors/styles.css

```diff
@@ -0,0 +1,50 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: aliceblue;
```

Comment:
The `background-color` property should be applied to the `section `rather than the `body`

Reply:
![image](https://user-images.githubusercontent.com/121365432/214656262-41f5e00d-f225-4621-b1cb-fc89e1a3ea08.png)